.github/workflows/performance.yml

```
@@ -0,0 +1,37 @@
+            fi
+            echo "Waiting for analyzer..."
+            sleep 2
+          done
+
+          echo "Analyzer failed to become ready"
+          docker compose logs analyzer
+          exit 1
+          
+      - uses: grafana/setup-k6-action@v1
+
+      - name: Run k6 analysis load test
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)